FirstSteps-Exercise.playground/Contents.swift

```diff
@@ -0,0 +1,214 @@
+// var a = 1
+// var b = 2
+// a = b // a will have the value 2. The original value of a is lost
+// b = a // b will remain the same
+
+// HINT 2
+// Use a third variable to save the original value of a.
+
+var aS = 1
+var bS = 2
+
+// your code here
+var c = aS
+aS = bS
+bS = c
+//////////////////////////////////////////////
+/////////         Last digit         /////////
+//////////////////////////////////////////////
+/*You are given a number a. Print the last digit of a.*/
+
+// HINT
+// Use the remainder % operator.
+// Remember that aLD = k * (aLD / k) + aLD % k
+// Can you think of a value for k that gives the last digit?
+
+var aLD = 123
+
+// your code here
+print(aLD % 100)
+
+/////////////////////////////////////////////
+/////////         Dog Years         /////////
+/////////////////////////////////////////////
+/*You are given Rocky’s age in dog years. Print Rocky’s age in human years. You know that 1 human year is 7 dog years.*/
+
+// HINT
+// Use division.
+
+var rockysAge = 50
+
+// your code here
+print(rockysAge / 7)
+
+////////////////////////////////////////////
+/////////         Brothers         /////////
+////////////////////////////////////////////
+/*Everyone hates solving word problems by hand so let’s make a program to solve them for us.
+ x years from now Alice will be y times older than her brother Bob. Bob is 12 years old. How many years does Alice have?*/
+
+// HINT
+// alice + xB = yB * (bob + xB)
+// Solve for alice
+
+var xB = 3
+var yB = 2
+var bob = 12
+
+// var alice = ?
+
+// your code here
+var alice = yB * (bob + xB) - xB
+
+//////////////////////////////////////////////////////
+/////////         Apples and Oranges         /////////
+//////////////////////////////////////////////////////
+/*You have x apples. Bob trades 3 oranges for 5 apples. He does not accept trades with cut fruit.
+ How many oranges can you get from Bob and how many apples will you have left?
+ 
+ The number of apples you will have left should be stored in a variable named apples. The number of oranges you will have after the trade should be stored in a variable named oranges.*/
+
+// HINT
+// Use the division(/) and the remainder(%) operator
+
+var xA = 17
+
+// your code here
+var oranges = xA % 3
+
+//////////////////////////////////////////////////
+/////////         Boys and Girls         /////////
+//////////////////////////////////////////////////
+/*A class consists of numberOfBoys boys and numberOfGirls girls.
+ Print the percentage of boys in the class followed by the percentage of girls in the class. The percentage should be printed rounded down to the nearest integer. For example 33.333333333333 will be printed as 33.*/
+
+// HINT 1
+// numberOfStudents … 100%
+// numberOfBoys … X%
+
+// HINT 2
+// numberOfStudents / 100 = numberOfBoys / X
+
+var numberOfBoys = 20.0
+var numberOfGirls = 60.0
+
+// your code here
+var percentBoys = Int(numberOfBoys / numberOfGirls * 100)
```

Comment:
in swift is not really need it to cast values Int(value), once numberOfBoys and numberOfGirls is set as Int automatically it will take the same type